src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/submission/SubmissionApiDelegateImpl.java

```diff
@@ -166,4 +169,13 @@ public EfilingError buildEfilingError(ErrorResponse errorResponse) {
         return response;
     }
 
+    private BigDecimal getFee(String type) {
+        try {
+            return efilingLookupService.getServiceFee(type).getFeeAmt();
+        } catch (DatatypeConfigurationException  e) {
```

Comment:
whats up with the red bar <- from 175 to 177

Reply:
Ahh we aren't testing the exception

Reply:
FOUND THE 1%